packages/stark-ui/src/modules/date-picker/components/date-picker.component.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, EventEmitter, ViewEncapsulation, Input, Output, ViewChild } from "@angular/core";
+import { MatDatepicker, MatDatepickerInput, MatDatepickerInputEvent } from "@angular/material";
+import * as moment from "moment";
+
+export type StarkDatePickerFilter = "OnlyWeekends" | "OnlyWeekdays" | ((date: Date) => boolean) | undefined;
+
+/**
+ * Component to display the stark date-picker
+ */
+@Component({
+	selector: "stark-date-picker",
+	templateUrl: "./date-picker.component.html",
+	encapsulation: ViewEncapsulation.None
+})
+export class StarkDatePickerComponent {
+	/**
+	 * id attribute of the form field wrapping the mat-datepicker
+	 * id attribute followed by "-input" of the mat-datepicker-input
+	 *
+	 */
+	@Input()
```

Comment:
It's difficult to understand the inputs available in the code because it is not sorted in alphabetic order.
Could you please sort your inputs / outputs (based on their name) ? 😊 (I think that will be needed also to generate the documentation correctly.